From 2d079f1ee22e317e0e60e90d52993d9a52480845 Mon Sep 17 00:00:00 2001
From: Zhenbo Li <litimetal@gmail.com>
Date: Tue, 1 Sep 2015 22:09:32 +0800
Subject: [PATCH] Hack for old autoconf

For example Centos 6
---
 aclocal.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aclocal.m4 b/aclocal.m4
index f7a12f1..ade2116 100644
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -33,7 +33,7 @@ AS_VAR_IF([ac_cv_prog_$1],[],
   [AS_VAR_IF([cross_compiling],[yes],[],
     [AS_UNSET([ac_cv_prog_$1])
      AC_CHECK_PROG([$1],[$2],[$2],[$3],[$4])])],
-[AS_VAR_COPY([$1],[ac_cv_prog_$1])])])
+[eval $1=\$ac_cv_prog_$1])])
 
 dnl **** Initialize the programs used by other checks ****
 dnl
-- 
2.1.2

