From 7e05e3276a5f62371c75c3c6523e79681f3928e2 Mon Sep 17 00:00:00 2001
From: Sebastian Lackner <sebastian@fds-team.de>
Date: Fri, 28 Feb 2014 19:37:48 +0100
Subject: configure: Replace AS_VAR_COPY with eval for old versions of
 autoconf

---
 aclocal.m4 |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aclocal.m4 b/aclocal.m4
index 8274b89..ea87730 100644
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -33,7 +33,7 @@ AS_VAR_IF([ac_cv_prog_$1],[],
   [AS_VAR_IF([cross_compiling],[yes],[],
     [AS_UNSET([ac_cv_prog_$1])
      AC_CHECK_PROG([$1],[$2],[$2],[$3],[$4])])],
-[AS_VAR_COPY([$1],[ac_cv_prog_$1])])])
+[eval $1=\$ac_cv_prog_$1])])
 
 dnl **** Initialize the programs used by other checks ****
 dnl
-- 
1.7.9.5

